src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php

```diff
@@ -0,0 +1,197 @@
+                'type' => 'bool',
+                'required' => false,
+            ];
+        }
+
+        return $description;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function filterProperty(string $property, $value, QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null)
+    {
+        if (
+            !isset($value[self::QUERY_PARAMETER_KEY]) ||
+            !$this->isPropertyEnabled($property) ||
+            !$this->isPropertyMapped($property, $resourceClass, true) ||
+            !$this->isNullableField($property, $resourceClass)
+        ) {
+            return;
+        }
+
+        if (in_array($value[self::QUERY_PARAMETER_KEY], ['true', '1', '', null], true)) {
+            $value = true;
+        } elseif (in_array($value[self::QUERY_PARAMETER_KEY], ['false', '0'], true)) {
+            $value = false;
+        } else {
+            $this->logger->notice('Invalid filter ignored', [
+                'exception' => new InvalidArgumentException(sprintf('Invalid value for "%s[%s]", expected one of ( "%s" )', $property, self::QUERY_PARAMETER_KEY, implode('" | "', [
+                    'true',
+                    'false',
+                    '1',
+                    '0',
+                ]))),
+            ]);
+
+            return;
+        }
+
+        $alias = 'o';
+        $field = $property;
+
+        if ($this->isPropertyNested($property)) {
+            list($alias, $field) = $this->addJoinsForNestedProperty($property, $alias, $queryBuilder, $queryNameGenerator);
+        }
+
+        $propertyParts = $this->splitPropertyParts($property);
+        $metadata = $this->getNestedMetadata($resourceClass, $propertyParts['associations']);
+
+        if ($metadata->hasAssociation($field)) {
+            if ($metadata->isCollectionValuedAssociation($field)) {
+                $queryBuilder
+                    ->andWhere(sprintf('%s.%s %s EMPTY', $alias, $field, $value ? 'IS NOT' : 'IS'));
+
+                return;
+            }
+
+            $queryBuilder
+                ->andWhere(sprintf('%s.%s %s NULL', $alias, $field, $value ? 'IS NOT' : 'IS'));
+
+            return;
+        }
+
+        if ($metadata->hasField($field)) {
+            $queryBuilder
+                ->andWhere(sprintf('%s.%s %s NULL', $alias, $field, $value ? 'IS NOT' : 'IS'));
+        }
+    }
+
+    /**
+     * Determines whether the given property refers to a nullable field.
+     *
+     * @param string $property
+     * @param string $resourceClass
+     *
+     * @return bool
+     */
+    protected function isNullableField(string $property, string $resourceClass): bool
+    {
+        $propertyParts = $this->splitPropertyParts($property);
+        $metadata = $this->getNestedMetadata($resourceClass, $propertyParts['associations']);
+
+        $field = $propertyParts['field'];
+
+        if ($metadata->hasAssociation($field)) {
```

Comment:
Look at my commit (made my changes in a separate commit), I refactored it a bit:

```diff

         if ($metadata->hasAssociation($field)) {
              if ($metadata->isSingleValuedAssociation($field)) {
 -                if ($metadata instanceof ClassMetadataInfo) {
 -                    $associationMapping = $metadata->getAssociationMapping($field);
 -
 -                    return $this->isAssociationNullable($associationMapping);
 -                } else {
 +                if (!($metadata instanceof ClassMetadataInfo)) {
                      return false;
                  }
 +
 +                $associationMapping = $metadata->getAssociationMapping($field);
 +
 +                return $this->isAssociationNullable($associationMapping);
              }
  
              return true;
          }
  
 -        if ($metadata->hasField($field)) {
 -            if ($metadata instanceof ClassMetadataInfo) {
 -                return $metadata->isNullable($field);
 -            }
 +        if ($metadata instanceof ClassMetadataInfo && $metadata->hasField($field)) {
 +            return $metadata->isNullable($field);
          }
```